### PR TITLE
Complete announcement-notification except for the auth part 

### PR DESCRIPTION
**Pull request by aenon0**

### CODE REVIEW

File: CustomEd.Classroom.Service/Consumers/StudentEvents/StudentUpdatedEventConsumer.cs [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 namespace CustomEd.Classroom.Service.Consumers
 {
+    ///I think you forgot to replace teacher with student
```
Comment (Amanuel94) [inline CustomEd.Classroom.Service/Consumers/StudentEvents/StudentUpdatedEventConsumer.cs:10]: What do you mean?

Comment (Amanuel94) [inline CustomEd.Classroom.Service/Consumers/StudentEvents/StudentUpdatedEventConsumer.cs:10]: What do you mean?

Comment (Amanuel94) [inline CustomEd.Classroom.Service/Consumers/StudentEvents/StudentUpdatedEventConsumer.cs:10]: What do you mean?

File: CustomEd.Classroom.Service/Model/Student.cs [hunk 2]
```diff
@@ -4,7 +4,7 @@ namespace CustomEd.Classroom.Service.Model;
 
 public class Student: BaseEntity
 {
-    public string? StudentId {get; set;}
+    public string? StudentId {get; set;} //This should be removed
```
Comment (Amanuel94) [inline CustomEd.Classroom.Service/Model/Student.cs:7]: This is the schoolId not the userID assigned by the database

File: CustomEd.Classroom.Service/appsettings.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
     "TokenLifetime": 60
   },
   "RabbitMQSettings":{
-    "Host":"localhost"
+    "Host":"amqps://xrbirzby:N2XAE5oEzUjXETriGu4rsxWlzhBuEfxz@kangaroo.rmq.cloudamqp.com/xrbirzby"
```
Comment (Amanuel94) [inline CustomEd.Classroom.Service/appsettings.json:21]: this should not be pushed. Add them to .gitignore

File: CustomEd.Classroom.Service/Consumers/StudentEvents/StudentUpdatedEventConsumer.cs (hunk 1 above)
Comment (aenon0) [reply CustomEd.Classroom.Service/Consumers/StudentEvents/StudentUpdatedEventConsumer.cs:10]: err1

Comment (aenon0) [reply CustomEd.Classroom.Service/Consumers/StudentEvents/StudentUpdatedEventConsumer.cs:10]: i meant student with teacher here

Comment (aenon0) [reply CustomEd.Classroom.Service/Consumers/StudentEvents/StudentUpdatedEventConsumer.cs:10]: same reason as the above two

File: CustomEd.Classroom.Service/Model/Student.cs (hunk 2 above)
Comment (aenon0) [reply CustomEd.Classroom.Service/Model/Student.cs:7]: Understood!

File: CustomEd.Classroom.Service/appsettings.json (hunk 3 above)
Comment (aenon0) [reply CustomEd.Classroom.Service/appsettings.json:21]: Sure I'll.

File: CustomEd.Classroom.Service/Consumers/StudentEvents/StudentUpdatedEventConsumer.cs (hunk 1 above)
Comment (aenon0) [reply CustomEd.Classroom.Service/Consumers/StudentEvents/StudentUpdatedEventConsumer.cs:10]: err2

**Pull request closed without merging**